dotcom-rendering/src/web/components/StickyBottomBanner/ReaderRevenueBanner.tsx

```diff
@@ -93,34 +100,14 @@ const buildPayload = async ({
 			),
 			countryCode,
 			weeklyArticleHistory: await asyncArticleCount,
-			optedOutOfArticleCount,
+			hasOptedOutOfArticleCount: optedOutOfArticleCount,
```

Comment:
this field name was wrong, so we haven't been sending this to the backend for some time.
It's probably ok because any clients who have opted out will not have an article count, so will not have seen messages containing article count

Reply:
having types shared with `support-dotcom-components` means we'll catch these issues in future